proto/utxorpc/v1alpha/watch/watch.proto

```diff
@@ -37,6 +37,7 @@ message WatchTxRequest {
 message AnyChainTx {
   oneof chain {
     utxorpc.v1alpha.cardano.Tx cardano = 1; // A Cardano transaction.
+    BlockRef block = 2; // Block containing the transaction
```

Comment:
I feel like maybe a field mask is better here than just returning a `BlockRef`, but I'm open to either

Reply:
I would put the `block` field outside the `oneof` so that we have the option of returning both the tx and the block when necessary. As you suggest, users can use the existing field mask in the request to specify what they want.

Reply:
ah yes, including it in the `oneof` was a mistake on my side. Fixed